src/db.py

```diff
@@ -31,15 +32,15 @@ def view_expenses(self) -> None:
             print('')
 
     def add_expense(
-        self,
-        date: str,
-        category: str,
-        amount: float,
-        description: str
+            self,
+            date: str,
+            category: str,
+            amount: float,
+            description: str
```

Comment:
This has weird indentation, you can undo the indentation here to what it originally was